### PR TITLE
Convert to Rmd manual - fix #201

### LINKED ISSUE #201: Make the Manual easy to update
Currently it seems to be in a raw html format: 
https://github.com/npct/pct-shiny/blob/master/static/FAQs.html

This is difficult for most people to contribute to.
The work is to convert it to a .md or preferable .Rmd (which works the same as .md) document?

### PR DESCRIPTION
**Pull request by Robinlovelace**
Please test @nikolai-b and @usr110 - want to close this soon

### CODE REVIEW

**Comment by nikolai-b** [comment]
Shiny already has bootstrap, is there a reason you are including it again here?  It looks like it'd end up being loaded twice.

File: regions_www/manual.html [hunk 1]
```diff
@@ -0,0 +1,436 @@
...
+<li>Census 2011 Reported Cycle Commuting</li>
+<li>Government Target: this scenario is based on the UK government’s proposed target (as set out in its draft <a href="https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/364791/141015_Cycling_Delivery_Plan.pdf">Cycling Delivery Plan</a>) to double cycling in England. Taking population increases and trip rate decreases into account, the target implies doubling the proportion of trips made by cycling nationwide. Trips that have a large distances or that already have a high rate of cycling will see cycling increase less than two-fold; trips with a below average current rate of cycling but high potential (based on the distance) will see cycling increase more than two-fold.</li>
+<li>Gender equality: Women have the same cycling rate as men</li>
+<li>Go Dutch: Cycling propensity at the levels seen in the Netherlands</li>
+<li>Electric bicycles: Willingness to cycle longer trips increases with widespread use of electric bicycles</li>
+</ol>
+</td>
+<td style="vertical-align: top; width: 535px;">
+<img src="/home/robin/npct/pct-shiny/www/assets/help/scenario-selection.png" alt="Scenario Selection" />
```
Comment (nikolai-b) [inline regions_www/manual.html]: This path won't work ^

File: regions_www/manual.html [hunk 2]
```diff
@@ -0,0 +1,436 @@
...
+<td style="vertical-align: top; width: 1054px;">
+<p><span style="font-weight: bold; text-decoration: underline;">Zone Attribute</span></p>
+<ol style="list-style-type: decimal">
+<li>Scenario Level of Cycling is the model predicted rate of cycling in the given area for the chosen scenario</li>
+<li>Scenario Increase in Cycling is the corresponding number of additional cycle commuters over and above Census 2011</li>
+<li>None: No zone attribute is displayed</li>
+</ol>
+</td>
+<td style="vertical-align: top; width: 535px;">
+<img src="/home/robin/npct/pct-shiny/www/assets/help/zone-attribute.png" alt="Zone Attribute" />
```
Comment (nikolai-b) [inline regions_www/manual.html]: or this one ^.  The links all seem to point to your local directory @Robinlovelace, sadly I've no Rmarkdown expertise so can't really suggest much, hope its an easy fix :smile:

File: static/manual-markdown.Rmd [hunk 3]
```diff
@@ -432,4 +421,15 @@ We plan to add more documentation to ease the creation of locally specific scena
...
 </div>
 
-<script>(function () { var script = document.createElement("script"); script.type = "text/javascript"; script.src = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"; document.getElementsByTagName("head")[0].appendChild(script); })();</script>
\ No newline at end of file
+<script>(function () { var script = document.createElement("script"); script.type = "text/javascript"; script.src = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"; document.getElementsByTagName("head")[0].appendChild(script); })();</script>
+
+```{r, echo=FALSE, eval=FALSE}
+# change image links
+f = readLines("regions_www/manual.html")
+f = gsub(pattern = "/home/robin/npct/pct-shiny/www/assets/", replacement = "./assets/", f)
```
Comment (nikolai-b) [inline static/manual-markdown.Rmd]: Ah, I see.
Sorry but this isn't a proper solution, it looks far hackier than just writing html.  Is there a reason we need R code in the manual?

**Comment by nikolai-b** [comment]
I don't want to be really down on this but from the looks of it changing to Rmarkdown is adding overhead and I want to clarify why its needed.  For me Rmarkdown makes sense on pages where we want to use R code but when we just want something static I don't see the advantage.  I know you don't like HTML so I guess that is an advantage but it does add a lot more complexity and @usr110 can't build it https://github.com/npct/pct-shiny/issues/248#issuecomment-220086076

File: static/manual-markdown.Rmd (hunk 3 above)
Comment (Robinlovelace) [inline static/manual-markdown.Rmd]: Fixed here: https://github.com/npct/pct-shiny/commit/efc0c66c6c8248c372f82e3b5542c95f0c1cd00e 

The wider issue of why to shift to markdown is addressed below - it's to allow people who don't want to write html, like @RachelAldred and me, to contribute to the manual.

**Comment by Robinlovelace** [comment]
@nikolai-b there are two issues. One was that the regex was file-system specific. I think that's been fixed (please test). The other one is should we shift to html to markdown at all? The answer is yes - as agreed with others (most recently @jf1 ), it's far more inviting to edit this than this.

It now builds into the right places automatically also and updates on the front page, providing a permalink for the manual (which has been requested many times). Final benefit of .Rmd: fexibility.

File: static/user-manual.html [hunk 4]
```diff
@@ -313,7 +308,7 @@ <h4 class="author"><em>PCT Team</em></h4>
 <tbody>
 <tr>
 <td align="center" valign="center">
-<a href="./assets/FAQs/Fig0.png" title="Figure 1: Proportion of trips cycled in England, according to the hilliness of the local area"><img src="./assets/FAQs/Fig0.png" alt="" /></a>
+<a href="./assets/FAQs/Fig0.png" title="Figure 1: Proportion of trips cycled in England, according to the hilliness of the local area"><img src="/home/robin/npct/pct-shiny/www/assets/FAQs/Fig0.png" alt="" /></a>
```
Comment (nikolai-b) [inline static/user-manual.html]: Still showing as robin ^

**Comment by nikolai-b** [comment]
Is it possible to remove bootstrap as this is included in shiny https://github.com/npct/pct-shiny/pull/250#issuecomment-220769231 ?

Like I said, if you can get it to work but this https://github.com/npct/pct-shiny/commit/efc0c66c6c8248c372f82e3b5542c95f0c1cd00e#diff-30c30f1ba3c3e88f11836f1ac6aedeccR434 and all the `R` code at the end isn't great, and now @jf1 and others have to install pandoc instead of just pushing an html change.  It also means you have to edit https://github.com/npct/pct-shiny/blob/d6d48735fa45ff800a511d07e3900e39888285dc/static/header-manual.html which is more html.

File: static/user-manual.html (hunk 4 above)
Comment (nikolai-b) [inline static/user-manual.html]: still showing as robin here ^

File: regions_www/manual.html [hunk 5]
```diff
@@ -0,0 +1,436 @@
...
+<p>So are the quietest routes for those groups, and the direct routes for others? No, because this project also shows that propensity to cycle declines more rapidly with distance for women and older people. If a quieter route creates a detour such that a 2 mile trip becomes effectively a 3 mile trip, younger men’s propensity to cycle the route will decrease 11%. But for younger women, the decline is 19%, and for older adults (60+) the propensity would decrease by 35%.</p>
+<p>This hits women’s and older men’s cycling twice: they are less likely than men to want to cycle the direct route, but then are also less likely to be willing to cycle an effectively longer alternative route. It’s likely to be part of the reason for disparities in cycling by age and gender.</p>
+<div style="margin: 0 auto; text-align: center">
+<table cellpadding="5" align="center" border="0" width="70%">
+<tbody>
+<tr>
+<td align="center" valign="center">
+<a href="./assets/FAQs/Route.png" title="Direct and Quiet Routes between two zones."><img src="/home/robin/npct/pct-shiny/www/assets/FAQs/Route.png" alt="" /></a>
```
Comment (nikolai-b) [inline regions_www/manual.html]: still showing as robin (I think, githuib might not have updated... :confused: )

**Comment by Robinlovelace** [comment]
Fair point - yes I think it's possible to re-use the bootstrap - will look to refactor and get back to you.

**Comment by Robinlovelace** [comment]
@nikolai-b I think the manual is fixed now - tested in pct-shiny app - pls confirm. Any pointers re. how to remove duplicate bootstrap - shouldn't be too tricky right? Thanks for the quickfire feedback.

**Comment by nikolai-b** [comment]
@Robinlovelace Merging this before getting a :+1: is out of process!  I know you wanted to check it on the test server but I can help do that if you need.  If there are outstanding comments then please don't merge.

**Comment by Robinlovelace** [comment]
Yes sorry about that - guess that's the issue with working at weekends. Anyway sure you'll agree the [result](http://pct.bike/manual.html) is pretty sweet and[ easy to edit](https://github.com/npct/pct-shiny/edit/master/static/manual.Rmd). Will look at your pct-load PR asap although may not be this eve.

**Pull request merged**